extensions/amp-lightbox/1.0/amp-lightbox.css

```diff
@@ -18,4 +18,18 @@ amp-lightbox {
   width: 100%;
   height: 100%;
   position: fixed;
+  visibility: hidden;
+  box-sizing: border-box;
+  background-color: rgba(0, 0, 0, 0.9);
```

Comment:
Weird that we have a background color with transparency. Are we ok with some very bright content seen through?

Reply:
I think so, given publishers can change this now. It should be somewhat translucent so end users know there is more content on the page than the lightbox modal they have open - but maybe not necessary if we do not allow it to be open on load. I was actually thinking 0.9 might be too opaque.